content/help/ops/security/repairing-citadel/index.md

```diff
@@ -5,6 +5,15 @@ weight: 10
 keywords: [security,citadel,ops]
 ---
 
+Istio currently supports only a **single Citadel instance**. Running multiple Citadel instances may introduce
```

Comment:
same warning here.  You could use a boilerplate or just C&P if you like.

Reply:
done.